### PR TITLE
[ADD] timesheet so_line protections

### PR DESCRIPTION
**Pull request by jjscarafia**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/sale/pull/585).

File: sale_timesheet_ux/models/account_analytic_line.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from odoo import models, fields
```
Comment (nicomacr) [inline sale_timesheet_ux/models/account_analytic_line.py]: no se usa `fields`

File: sale_timesheet_ux/__manifest__.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
     ],
```
Comment (nicomacr) [inline sale_timesheet_ux/__manifest__.py:11]: falta cambiar la version https://github.com/ingadhoc/sale/pull/585/files#diff-24037374e0934b65819fd5c43db8731e3cce334aff27143494ce0c9e60259d6dR4

File: sale_timesheet_ux/README.rst [hunk 3]
```diff
@@ -15,6 +15,21 @@ Sales Timesheet UX
 ==================
 
 * When confirming a sale order with service products with Service tracking = "Create a task in sales order's project", if the SO has an analytic account that is related to ONE project, then we don't create a project but instead link the new tasks to this project.
+* Incorporar protección de cambios en partes de hora:
+a) Odoo, mediante distintos mecanismos, puede modificar la sale line vinculada a una linea de parte de horas.(campo so_line de account.analytic.line). Esto pasa si cambia
+   1. billiable de proyecto
+   2. empleado
+   3. la linea de venta en tarea, ticket o proyecto asociados al parte de horas. Estos a su vez cambian si:
+      3.1 en ticekts, si cambia: commercial partner, team_id.use_helpdesk_sale_timesheet, project_id.pricing_type o project_id.sale_line_id
```
Comment (nicomacr) [inline sale_timesheet_ux/README.rst]:
```suggestion
   3. la linea de venta en tarea, ticket o proyecto asociados al parte de horas. Estos a su vez cambian si:
      3.1 en tickets, si cambia: commercial partner, team_id.use_helpdesk_sale_timesheet, project_id.pricing_type o project_id.sale_line_id
```

**Review by nicomacr — CHANGES_REQUESTED** [review]

**Comment by jjscarafia** [comment]
@roboadhoc r+

**Comment by jjscarafia** [comment]
@roboadhoc override=ci/oba_without_tests

**Comment by jjscarafia** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward.

**Pull request closed without merging**